src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,60 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
```

Comment:
```suggestion
    private static final int MAX_SIZE = 10;
```
it's not to have 2 variables with the same names